env_setup_acc.sh

```diff
@@ -1,42 +1,68 @@
-mkdir PLUGIN_DEP_INSTALL
-export DEPS_INSTALL=$PWD/PLUGIN_DEP_INSTALL
-mkdir PLUGIN_DEP_SRC
-export SRC_DIR=$PWD/PLUGIN_DEP_SRC
+#!/bin/bash
+#
+# Copyright 2023 Intel Corporation
+# SPDX-License-Identifier: GPL-3.0-only
+#
+# Script to setup environment variables facilitating IPsec-Recipe compilation
+#
+
+set -e
+
+# Define directories
+DEPS_INSTALL_DIR="PLUGIN_DEP_INSTALL"
```

Comment:
1. Variables ending in `_DIR` are usually directory paths, not bare directory names.
2. I also see no reason for assigning the bare directory name to a variable, using that to define a second variable, and never using the first variable again.

```bash
DEPS_INSTALL=$PWD/PLUGIN_DEP_INSTALL
SRC_DIR=$PWD/PLUGIN_DEP_DIR

mkdir -p ${DEPS_INSTALL}
mkdir -p ${SRC_DIR}
```
Just define one variable for each and use it throughout the script.